test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -150,9 +156,18 @@ def test_single_mom_hook_failure_affects_vnode(self):
         and check if the job runs on one of the vnodes.
         """
         single_mom = self.moms.values()[0]
+        start_time = int(time.time())
         self.create_multi_vnodes(1)
         self.create_mom_hook()
 
+        # Check if hook files were copied to mom
+        single_mom.log_match(
+            "h1.HK;copy hook-related file request received",
+            existence=True, starttime=start_time, interval=2)
+        single_mom.log_match(
+            "h1.PY;copy hook-related file request received",
+            existence=True, starttime=start_time, interval=2)

```

Comment:
Consider removing existence=True.  It is the default (and the other log_match() calls).

Reply:
done